spring-xd-lang-detector/README.md

```diff
@@ -0,0 +1,104 @@
+
+## Code Tour
+
+The heart of the sample is the processing module named LanguageDetectionProcessor.java.  This uses the langdetect API to detect the language of a text property in the given input.  The tuple data type is used as a generic container for keyed data.
+
+
+## Building with Maven
+
+	$ mvn clean package
+
+## Building with Gradle
+
+	$./gradlew clean test bootRepackage
+
+## Using the Custom Module
+
+The uber-jar will be in `[project build dir]/xd-langdetect-1.0.0.BUILD-SNAPSHOT.jar`. To install and register the module to your Spring XD distribution, use the `module upload` Spring XD shell command. Start Spring XD and the shell:
+
+
+	_____                           __   _______
+	/  ___|          (-)             \ \ / /  _  \
+	\ `--. _ __  _ __ _ _ __   __ _   \ V /| | | |
+ 	`--. \ '_ \| '__| | '_ \ / _` |  / ^ \| | | |
+	/\__/ / |_) | |  | | | | | (_| | / / \ \ |/ /
+	\____/| .__/|_|  |_|_| |_|\__, | \/   \/___/
+    	  | |                  __/ |
+      	|_|                 |___/
+
+	eXtreme Data
+	1.1.0.BUILD-SNAPSHOT | Admin Server Target: http://localhost:9393
+	Welcome to the Spring XD shell. For assistance hit TAB or type "help".
+
+	xd:>module upload --file [path-to]/xd-langdetect-1.0.0.BUILD-SNAPSHOT.jar --name langdetect --type processor
+	Successfully uploaded module 'processor:langdetect'
+
+The module configuration:
+
+    xd:>module info --name processor:langdetect
+    Information about processor module 'langdetect':
+
+      Option Name                 Description                                                                                         Default          Type
+      --------------------------  --------------------------------------------------------------------------------------------------  ---------------  ----------
+      langOutput                  The name of the output property the detected language is written to.                                pred_lang        String
+      textInput                   The name of the property that contains the input text.                                              text             String
+      languageProfileLocation     The location of the language model. If empty we fall back to the profiles shipped with langdetect.                   String
+      textModel                   The name of the text model that should be used either SHORTTEXT or LONGTEXT.                        SHORTTEXT        String
+      returnLanguageProbabilites  Returns the detected language probabilities if enabled.                                             false            Boolean
+      returnMostLikelyLanguage    Returns the most likely detected language if enabled.                                               true             Boolean
+      langProbOutput              The name of the output property the detected language probabilities are written to.                 pred_lang_probs  JSONObject
```

Comment:
This table is not in synch with the properties you commited. Also, not sure about JSONObject